CHANGELOG.md

```diff
@@ -5,32 +5,106 @@ All notable changes to this project will be documented in this file.
+and [SIP-20](https://github.com/obycode/sips/blob/bitwise-ops/sips/sip-020/sip-020-bitwise-ops.md).
+
+The changelog for this release is a high-level summary of these SIPs.
 
 ### Added
 
-- Clarity function `stx-transfer?` now takes a 4th optional argument, which is a memo.
-- Added a new parser which will be used to parse Clarity code beginning with 2.1,
-  resolving several bugs in the old parser and improving performance.
-- Documentation will indicate explicitly which Clarity version introduced each
-  keyword or function. 
-- Clarity2 improvements to traits (see #3251 for details):
+- There is a new `.pox-2` contract for implementing proof-of-transfer.  This PoX
+  contract enables re-stacking while the user's STX are locked, and incrementing
+the amount stacked on top of a locked batch of STX.
+- The Clarity function `stx-account` has been added, which returns the account's
+  locked and unlocked balances.
+- The Clarity functions `principal-destruct` and `principal-construct?`
+  functions have been added, which provide the means to convert between a
+`principal` instance and the `buff`s and `string-ascii`s that constitute it.
+- The Clarity function `get-burn-block-info?` has been extended to support
+  fetching the burnchain header hash of _any_ burnchain block starting from the
+sortition height of the Stacks genesis block, and to support fetching the PoX
+addresses and rewards paid by miners for a particular burnchain block height.
+- The Clarity function `slice` has been added for obtaining a sub-sequence of a
+  `buff`, `string-ascii`, `string-utf8`, or `list`.
+- Clarity functions for converting between `string-ascii`, `string-utf8`,
+  `uint`, and `int` have been added.
+- Clarity functions for converting between big- and little-endian
+`buff`  representations of `int` and `uint` have been added.
+- The Clarity function `stx-transfer-memo?` has been added, which behaves the
+  same as `stx-transfer?` but also takes a memo argument.
+- The Clarity function `is-standard` has been added to identify whether or not a
+  `principal` instance is a standard or contract principal.
+- Clarity functions have been added for converting an arbitrary Clarity type to
+  and from its canonical byte string representation.
+- The Clarity function `replace-at?` has been added for replacing a single item
+  in a `list`, `string-ascii`, `string-utf8`, or `buff`.
+- The Clarity global variable `tx-sponsor?` has been added, which evaluates to
+  the sponsor of the transaction if the transaction is sponsored.
+- The Clarity global variable `chain-id` has been added, which evaluates to the
+  4-byte chain ID of this Stacks network.
+- The Clarity parser has been rewritten to be about 3x faster than the parser in
+  Stacks 2.05.x.x.x.
+- Clarity trait semantics have been refined and made more explicit, so as to
+  avoid certain corner cases where a trait reference might be downgraded to a
```

Comment:
Maybe "principal" was supposed to be at the end of this sentence?

Reply:
Fixed